Procfile

```diff
@@ -1 +1,2 @@
 web: gunicorn wsgi:application
+worker: celery -A fossevents worker -l info --concurrency=2 -B
```

Comment:
are we running celery beat as well? won't it have issues with multiple workers http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#starting-the-scheduler